packages/data-stores/src/plans/reducer.ts

```diff
@@ -30,11 +28,6 @@ const DEFAUlT_STATE: {
 
 const reducer = function ( state = DEFAUlT_STATE, action: PlanAction ) {
 	switch ( action.type ) {
-		case 'SET_PLAN':
-			return {
-				...state,
-				selectedPlanSlug: action.slug,
-			};
```

Comment:
With this state moved to consumer app's store (`Onboard` at the moment) it will be easier to understand the scope of `plans` / `domain-suggestions` data-stores. Their purpose is to store data fetched from the API and error/loading related states. Keeping track of application state (plan, domain, last search query, etc) should be the responsibility of each consumer app.
Some context about this: https://github.com/Automattic/wp-calypso/pull/43858/files#r449029095

Reply:
> Keeping track of application state (plan, domain, last search query, etc) should be the responsibility of each consumer app.

Agree! This makes it clear who does what now.